src/main/cpp/src/get_json_object.cu

```diff
@@ -838,6 +833,31 @@ rmm::device_uvector<path_instruction> construct_path_commands(
   return cudf::detail::make_device_uvector_sync(path_commands, stream, mr);
 }
 
+__device__ thrust::pair<bool, size_t> get_json_object_size_single(
+  char const* input,
+  cudf::size_type input_len,
```

Comment:
How about `device_span<char const>`?

Reply:
Oh wait, why do we have this overload of `get_json_object_size_single`? Just one version is not enough?

Reply:
Yes it won't gain performance, reverted.